documentation/modules/auxiliary/scanner/dcerpc/windows_deployment_services.md

```diff
@@ -0,0 +1,40 @@
+## Vulnerable Application
+
+This module retrieves the client unattend file from Windows Deployment Services RPC service and parses out the stored credentials. Tested against Windows 2008 R2 x64 and Windows 2003 x86.
+
+More information can be found on the [Rapid7 Vulnerability & Exploit Database page](https://www.rapid7.com/db/modules/auxiliary/scanner/dcerpc/windows_deployment_services) and pull request #1420 (https://github.com/rapid7/metasploit-framework/pull/1420).
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Do: `use modules/auxiliary/scanner/dcerpc/windows_deployment_services`
+  3. Do: `show options`
+    ...show and set options...
+  4. Do: `run`
+
+## Scenarios
+
```

Comment:
same as before, add L3 header with what version of windows you tested this against